Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+source 'https://rubygems.pkg.github.com/art19'
```

Comment:
```suggestion

# pull the ART19 fork of cassandra-driver for CI
source 'https://rubygems.pkg.github.com/art19' do
  gem 'cassandra-driver', '~> 3.5.2.1'
end
```

Reply:
No opposed, I did think it would be weird given the gemspec import